docs/SemanticARC.md

```diff
@@ -376,12 +434,19 @@ generalized dominators that can be used. If computation of generalized
 dominators is too expensive for normal use, they could be used on specific
 verification bots and used when triaging bugs.
 
+[EE]
+When looking at semantic ARC in terms of local RCs (see my comment above), the verifier is very simple. It's a dataflow algorithm, which ensures that the local RCs are consistent at block entries for all RC roots. It's similar to what we do to check proper nesting of alloc_stacks.
+
+Beside that, I didn't understand what algorithm you propose (what is "disproving joint dominance")?
```

Comment:
> That implies proving that the consumers of the value do not have any paths in between them 

Sure, that's what the data flow algorithm would do.

> post-dominate

Yeah, if not, a +1 value would reach the exit block in the data flow

> the retain set

What is the retain set? Isn't there only a single producer (SSA).